PrebidMobile/Rendering/PrebidMobileDemoRendering/PrebidMobileDemoRendering/PrebidMobileDemoRendering/ViewControllers/Configuration/NativeAdConfigEditor/EditorUtils.swift

```diff
@@ -0,0 +1,14 @@
+//
+//  EditorUtils.swift
+//  PrebidMobileDemoRendering
+//
+//  Copyright © 2021 Prebid. All rights reserved.
+//
+
+import Foundation
+
+class EditorUtils{
```

Comment:
Can we make this functionality as an extension for NativeAsset?

Reply:
Done